spec/factories.rb

```diff
@@ -12,4 +12,5 @@
     email 'email@email.com'
     password '123456'
   end
+
```

Comment:
👾 <- space invader aka "get rid of the whitespace"

Reply:
fixed